hls4ml/converters/__init__.py

```diff
@@ -303,6 +301,7 @@ def convert_from_pytorch_model(
 
     model_config = hls_config.get('Model', None)
     config['HLSConfig']['Model'] = _check_model_config(model_config)
+    config['InputShape'] = hls_config.get('InputShape', None)
```

Comment:
I think we shouldn't have a default of `None` for input shape because this will propagate further and then lead to errors which won't make it clear what is the original cause

Reply:
Good point, we're now raising an exception if that parameter is not found. No point in continuing.